mdfreader/mdf.py

```diff
@@ -152,8 +154,8 @@ def add_channel(self, dataGroup, channel_name, data, master_channel, \
         """
         if channel_name in self:
             if self[channel_name][dataField] is not None:
-                channel_name += '_' + str(dataGroup)
-                self[channel_name] = {}
+            channel_name += '_' + str(dataGroup)
```

Comment:
Yes absolutely it is necessary -- just was wondering if I *removed* that indent.  Seems like it is just a merge issue.  Merging with git + Python is a little more tricky...  If this were C it won't be a non-issue.